app/assets/stylesheets/footer.css

```diff
@@ -15,14 +15,14 @@
  */
 
  .footer {
-   position: absolute;
+   position: fixed;
```

Comment:
@adamjohnsnow could we maybe move this back to absolute? It's a bit annoying to have the footer fixed on screen when you're scrolling down - especially on a small screen.